jekyll/_cci2/language-android.md

```diff
@@ -9,20 +9,37 @@ order: 9
 
 *[Tutorials & 2.0 Demo Apps]({{ site.baseurl }}/2.0/tutorials/) > Language Guide: Android*
 
-This guide will help you get started with an Android application on CircleCI.
+This document describes
+how to set up an Android project on CircleCI
+in the following sections.
 
 * TOC
 {:toc}
 
 ## Overview
 
-There are some assumptions made in this guide:
+This guide provides an introduction to Android development on CircleCI.
+If you are looking for a `.circleci/config.yml` template for Android,
+see the [Sample Configuration](#sample-configuration) section of this document.
 
-- We assume that your Android project is built with `gradle` (this is the default for projects created with [Android Studio](https://developer.android.com/studio).
-- We assume that your project is located in the root of your `git` repository, with the application located in a subfolder named `app`.
+**Note:**
+Due to how CircleCI implements virtualization on Linux,
+running the Android emulator is not supported.
+To run emulator tests from a job,
+consider using an external service like [Firebase Test Lab](https://firebase.google.com/docs/test-lab).
+For more details,
+see the [Testing With Firebase Test Lab](#testing-with-firebase-test-lab) section below.
 
-Running the Android emulator is not currently supported by the type of virtualization CircleCI 2.0 uses on Linux. To run emulator tests from a job, consider using an external service like [Firebase Test Lab](https://firebase.google.com/docs/test-lab). To do this, you can use the [gcloud command line](https://firebase.google.com/docs/test-lab/command-line). Google Cloud tools are preinstalled in our Android Docker images, which you can find on [GitHub](https://github.com/CircleCI-Public/circleci-dockerfiles/tree/master/android/images) or [Docker Hub](https://hub.docker.com/r/circleci/android/tags).
```

Comment:
- Note about unsupported emulation has been moved up to the **Overview**.
- The information about Google Cloud tools has been moved down to the new **Testing with Firebase Test Lab** section.

Reply:
I don't like this rewrite, 'due to how' is awkward and not clear, so please revert this change back to the original that was already reviewed and approved when I added it.